app/views/shared/_nav.html.erb

```diff
@@ -46,6 +46,7 @@
 
         <ul class="usa-nav-submenu" id="nav-resources" aria-hidden="true">
           <li><%= link_to 'Helpful Links', page_path('resources') %></li>
+          <li><%= link_to 'PPM Estimator', ppm_estimator_path %></li>
```

Comment:
let's also add this to the 404 page. i'm pretty sure there are also other ppm estimator #TODO links

Reply:
done.